src/pages/home/Home.jsx

```diff
@@ -1,5 +1,57 @@
+import { useDocTitle } from "../../hooks/useDocTitle";
 import "./home.css";
-
+import images from "../../assets/images";
+import { useDispatch, useSelector } from "react-redux";
```

Comment:
suggestion : you can use absolute imports

Reply:
will look into it